packages/react-app/src/components/EthbotLearn/EthbotLearn.jsx

```diff
@@ -86,6 +87,14 @@ export default function EthbotLearn() {
 			<div className="bg-bot5 bg-auto bg-no-repeat bg-contain w-full ethbotlearn-height">
 				{interior}
 			</div>
+		: botNum === 5 ?
```

Comment:
Consider using Switch statement for this instead

Reply:
Or even better, use `bg-bot${botNum + 1}` in the className depending on your use case

Reply:
I am adding changes for the switch statement